ndcube/asdf/resources/schemas/ndcube-0.1.0.yaml

```diff
@@ -16,9 +16,9 @@ properties:
   wcs:
     anyOf:
       - tag: "tag:stsci.edu:gwcs/wcs-1.*"
-      - tag: "tag:sunpy.org:ndcube/resampledwcs-0.1.0"
-      - tag: "tag:sunpy.org:ndcube/reorderedwcs-0.1.0"
-      - tag: "tag:sunpy.org:ndcube/compoundwcs-0.1.0"
+      - tag: "tag:sunpy.org:ndcube/resampledwcs-*"
```

Comment:
@Cadair This seems to have broken the tests.

Reply:
I believe the test is not failing because of the changes related to asdf, so we can retain the changes.